.travis.yml

```diff
@@ -146,10 +142,10 @@ after_success:
     fi
 
   # The below script used to be run as post-processor for the Openstack's builder.
-  # In order to avoid running Terraform within Packer, hence getting a cumbersome and tedious log output. travis_retry
+  # In order to avoid running Terraform within Packer, hence getting a cumbersome and tedious log output.
   - >
     if [ $HOST_CLOUD = 'openstack' ]; then
-        bin/os_pp.sh
+        travis_retry bin/os_pp.sh
```

Comment:
I forgot that this script runs terraform, hence I've put back the `travis_retry` flag.